### DISCUSSION TITLE
Feedback thread for Favorite Pipelines

### DISCUSSION THREAD

**Discussion by matthewpereira** [concourse/concourse#6028] [Show and tell]
**TLDR: This thread is to gather feedback for the use of the Favorite Pipelines feature released with v6.5.0.**

## Code changes

If you're interested in seeing the code changes that contributed to this functionality, this issue should lead you to all relevant commits and PRs:
#5434 - Favorite Pipelines

## What's it look like? How does it work?

<img src="https://user-images.githubusercontent.com/6372810/79272600-adc41580-7e6f-11ea-9bf2-d4e536a96de9.png" alt="Pipeline Tile" width="240">

With the release of v6.5.0, pipeline tiles gain a star icon in the lower right that allows users to toggle their status as a favorite pipeline on and off. This ability also exists in the sidebar, with a star icon on each pipeline listed there, and in the top bar when viewing an individual pipeline.

![Favorite Pipeline SS-1](https://user-images.githubusercontent.com/6372810/91737194-c150ee00-eb7c-11ea-9086-41c18ea7f260.png)

When a user marks a pipeline as a favorite, it joins any other favorite pipelines in a special area at the top of the CI monitor dashboard, as well as the top of the sidebar. This means that these favorite pipelines are immediately visible and accessible as soon as Concourse loads, no longer requiring any scrolling, searching, or filtering to find. The highlighted star also makes it easier to spot these pipelines in their usual context further down in the scroll.

For now, your favorites are saved locally, in your browser. This means that if you switch browsers, workstations, or clear your local storage, you won't see the favorite selections persist.

## What’s next?

We have a few ideas for further upgrading the sidebar that we'll detail in the coming weeks, but the Concourse team would love to get any feedback the community has in the meantime so that our next iteration is as powerful as possible - and to hear whether or not our ideas for future iterations are on the right track. What's working for you?
